src/Navigation/Education/EducationContent.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./index.css";
+function EducationContent(props) {
```

Comment:
Suggestion: Can de-structure `props` object to expose `content` and use it directly inside, no need to do `props.content`
`function EducationContent(props)`  => `function EducationContent({ content }) `

Reply:
also, arrow functions are getting more popular these days, cuz of binding issue.
It would be good idea to write arrow functions now onwards like:

`const EducationContent = ({ content }) => {`

Reply:
https://dev.to/sarah_chima/object-destructuring-in-es6-3fm